README.md

```diff
@@ -85,6 +113,27 @@ yarn test:a11y
 
 Make sure all the paths in your application are covered by accessibility tests (see [a11y.ts](src/test/a11y/a11y.ts)).
 
+#### Cross browser tests
+
+Cross browser tests are run via [CodeceptJS](https://codecept.io/) using the [Playwright](https://playwright.dev/) plugin which tests Chrome/Edge, Safari and Firefox and [WebDriver](https://www.w3.org/TR/webdriver/) plugin which tests Internet Explorer via [Sauce Labs](https://saucelabs.com/).
+
+Cross browser tests are automatically run via the nightly Jenkins pipeline.
+
+Running cross browser tests locally, start the server then run:
+
+```bash
+# Playwright - Chrome/Edge, Safari and Firefox
+TEST_HEADLESS=false yarn test:crossbrowser:latest
+
+# WebDriver - Internet Explorer (via Sauce Labs)
+brew install sauce-connect
+export SAUCE_USERNAME="your-username"
+export SAUCE_ACCESS_KEY="token-from-sauce-labs-settings-page"
+export SAUCE_TUNNEL_IDENTIFIER="$SAUCE_USERNAME-tunnel"
+sc -u $SAUCE_USERNAME -k $SAUCE_ACCESS_KEY -x https://eu-central-1.saucelabs.com/rest/v1 -i $SAUCE_TUNNEL_IDENTIFIER -B all
+yarn test:crossbrowser:ie
```

Comment:
maybe cram all this into a .sh file

Reply:
👍 I've updated the details in the readme to make this clearer in bfd8b64